Module#02/model/src/main/java/com/epam/esm/entity/Certificate.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.esm.entity;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.io.Serializable;
+import java.math.BigDecimal;
+import java.time.Instant;
+import java.util.Set;
+
+@Data
+@Builder
+public class Certificate implements Serializable {
+
+    private Long id;
+    private String name;
+    private String description;
+    private BigDecimal price;
+    private Instant createDate;
+    private Instant lastUpdateDate;
+    private Integer duration;
+    private Set<Tag> tags;
```

Comment:
@EqualsAndHashCode.Exclude - please, read more about it and apply it to tags

Reply:
Added ToString.Exclude and EqualsAndHashCode.Exclude avoiding nested cause any stack overflow errors due to circular references